src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -4841,7 +4843,17 @@ def webapp_up(cmd, name=None, resource_group_name=None, plan=None, location=None
 
     if logs:
         _configure_default_logging(cmd, rg_name, name)
-        return get_streaming_log(cmd, rg_name, name)
+        # ping the site so that the site container is ready
+        # logsteam does not work if the site container has not started
+        # required for https://github.com/Azure/azure-cli/issues/23058
+        _url = _get_url(cmd, rg_name, name)
+        logger.warning("Waiting for the app to be ready ...")
```

Comment:
why is this a warning? seems like an info to me

Reply:
only warning logs get printed by default for the user. This seems to be the convention for the repo.